test_user_management.py

```
@@ -0,0 +1,391 @@
+        # SECURITY ISSUE 2: SQL injection vulnerability - using string formatting
+        cursor.execute(f"""
+            CREATE TABLE IF NOT EXISTS users (
+                id INTEGER PRIMARY KEY AUTOINCREMENT,
+                username TEXT UNIQUE NOT NULL,
+                password TEXT NOT NULL,
+                email TEXT,
+                role TEXT DEFAULT 'user',
+                is_active BOOLEAN DEFAULT 1,
+                created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+            )
+        """)
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)